app/css/views/themes.css

```diff
@@ -42,4 +42,5 @@
   margin: 0 0 0 1rem;
   padding: 0;
   right: 0;
+  font-size: 0;
```

Comment:
I added the x because of a flash of unloaded image, but the `font-size` should be sufficient.